packages/bpk-component-bottom-sheet/src/BpkBottomSheet.tsx

```diff
@@ -15,86 +15,145 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { useState, type ReactNode } from "react";
+import type { ReactNode } from "react";
+import { useState } from "react";
 
-import { Portal, cssModules } from "../../bpk-react-utils";
-import { withScrim } from "../../bpk-scrim-utils";
+import BpkBreakpoint, { BREAKPOINTS } from "../../bpk-component-breakpoint";
+// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
+import { BpkButtonLink } from '../../bpk-component-link';
+// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
+import BpkCloseButton from '../../bpk-component-close-button';
+import { BpkDialogWrapper, cssModules } from "../../bpk-react-utils";
+import BpkNavigationBar from "../../bpk-component-navigation-bar";
 
 import STYLES from './BpkBottomSheet.module.scss';
-import BpkBottomSheetInner from "./BpkBottomSheetInner";
-
 
 const getClassName = cssModules(STYLES);
-const ScrimBpkBottomSheetInner = withScrim(BpkBottomSheetInner)
 
 export type Props = {
   actionText?: string;
+  ariaLabelledby: string;
   children: ReactNode;
   closeLabel?: string;
   closeOnEscPressed?: boolean;
   closeOnScrimClick?: boolean;
   id: string;
-  isOpen: boolean;
   onAction?: () => void;
   onClose: (
-    arg0?: TouchEvent | MouseEvent | KeyboardEvent,
+    arg0?: Event | KeyboardEvent | MouseEvent | PointerEvent | TouchEvent,
     arg1?: {
       source: 'ESCAPE' | 'DOCUMENT_CLICK';
     },
   ) => void;
   title?: string;
   wide?: boolean;
-  getApplicationElement: () => HTMLElement | null;
```

Comment:
wouldn't removing these props make this into a breaking change? 🤔

Reply:
You're right. This throws a ts error when having the property in use but not expected. I'll change the label on this PR to major.

Surprisingly though running the storybook with that extra prop doesn't cause any breakages or even console.logs. Still not good to keep it in, but something interesting to know.